Makefile

```diff
@@ -52,16 +53,19 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen kustomize ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	$(KUSTOMIZE) build config/kustomize-to-helm/overlays/crds -o charts/moco/crds
+	$(KUSTOMIZE) build config/crd -o config/crd/tests
+	$(KUSTOMIZE) build config/kustomize-to-helm/overlays/crds -o charts/moco/templates/generated/crds
 	$(KUSTOMIZE) build config/kustomize-to-helm/overlays/templates > charts/moco/templates/generated/generated.yaml
 
 .PHONY: generate
-generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: controller-gen conversion-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONVERSION_GEN) -i ./api/v1beta2/ -o . -O zz_generated.conversion --go-header-file hack/boilerplate.go.txt
```

Comment:
Use `conversion-gen` to generate the conversion code.

Reply:
@d-kuro 
Could you tell me if this is needed?
Because kubebuilder book does not mention this tool, I'm not sure using this is the right way or not.
https://book.kubebuilder.io/multiversion-tutorial/conversion.html

Reply:
I don't think it's necessary, but it makes sense to use conversion-gen because writing conversion code for an Object with many fields is tedious and can lead to mistakes.

conversion-gen is a proven tool that is also used in k/k and cluster-api, so I think no problem in using it.

Reply:
Understood.  Thank you for your explanation.